src/Microsoft.ML.ImageAnalytics/ImageResizer.cs

```diff
@@ -399,7 +394,6 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                         }
                         else
                             dst = new Bitmap(info.ImageWidth, info.ImageHeight, src.PixelFormat);
-
```

Comment:
It looks like you deleted the blank line following this line. The standard that has been followed in ML.NET is to leave single line without braces. I think that blank was to avoid the confusion with the block below. 
Please retain that blank line (or add braces around the single line)